.gitignore

```diff
@@ -0,0 +1,6 @@
+# .gitignore
+node_modules/
+
```

Comment:
- [ ] Remember that when we create a Pull Request, the person that it's going to review your project needs to know on the first view the changes that you're doing on that PR. Kindly add a description of the changes that you are making on this PR, you can use the requirements of this activity for help 💯